saas/templates/saas/base_dashboard.html

```diff
@@ -25,62 +89,7 @@
     $interpolateProvider.startSymbol('[[');
     $interpolateProvider.endSymbol(']]');
 }]);
-reopenSaasApp.constant('settings', {
```

Comment:
Yeah, it's quite ugly, but we can get rid of it as per the previous comment you made.